manifests/init.pp

```diff
@@ -70,11 +63,23 @@
 
   case $facts['os']['family'] {
     'RedHat': {
+      $os_package_require = []
     }
     'Debian': {
+      $os_package_require = [Class['::apt::update']]
     }
     default: {
       fail("Detected osfamily <${::osfamily}>. Only RedHat and Debian are supported.")
```

Comment:
I dont think osfamily is used anymore in lieu of `$facts['os']['family']`

Reply:
Changed to match how fact is referenced for actual case statement to use `$facts['os']['family']`.